lib/ClimaCoreTempestRemap/src/netcdf.jl

```diff
@@ -49,10 +49,69 @@ the type used by [`remap_weights`](@ref).
 
 If a compatible dimension already exists, it will be reused.
 """
-function def_space_coord(
+def_space_coord(
     nc::NCDataset,
     space::Spaces.SpectralElementSpace2D;
     type = "dgll",
+) = def_space_coord(nc, space, space.topology.mesh; type)
+
+function def_space_coord(
+    nc::NCDataset,
+    space::Spaces.SpectralElementSpace2D,
+    ::Meshes.RectilinearMesh;
+    type = "dgll",
+)
+    if type == "cgll"
+        nodes = Spaces.unique_nodes(space)
+    elseif type == "dgll"
+        nodes = Spaces.all_nodes(space)
+    else
+        error("Unsupported type: $type")
+    end
+    ncol = length(nodes)
+
+    if haskey(nc, "Y")
+        # dimension already exists: check correct size
+        if size(nc["Y"]) != (ncol,)
+            error("incompatible horizontal dimension already exists")
+        end
+        return (nc["X"], nc["Y"])
+    end
+
+    # # dimensions
+    defDim(nc, "ncol", ncol)
+
+    # variables
+    ## X
+    X = defVar(nc, "X", Float64, ("ncol",))
+    X.attrib["units"] = "degrees_east"
+    X.attrib["axis"] = "X"
+    X.attrib["long_name"] = "x"
+    X.attrib["standard_name"] = "x"
```

Comment:
Ah, forgot to grab that. So, no, unless I got lucky.

Reply:
Found the ref [here](https://cfconventions.org/cf-conventions/cf-conventions.html#_two_dimensional_latitude_longitude_coordinate_variables)

Reply:
Alright, updated. I didn't see an example for `standard_name` in the reference, but the other examples seem to just set the standard name equal to the long name, so that's what I did here.

Reply:
Does this look okay, cc @simonbyrne

Reply:
standard names are here:
https://cfconventions.org/Data/cf-standard-names/79/build/cf-standard-name-table.html

I'm not sure there is an appropriate one: maybe @jiahe23 or @szy21 might know?

Reply:
Yes, this is the cf standard. Not sure if we have to follow it though. Do we need `standard_name`?

Reply:
I've never used it.. Also, I don't see the x-coordinate in that list

Reply:
> I've never used it.. Also, I don't see the x-coordinate in that list

It has longitude, if this is what you meant by x-coordinate

Reply:
My personal taste prefers the the ECMWF varnames, but I am fine with any standard.

Reply:
I guess so? Should we still use lat-long for the rectangle domain?

Reply:
I have no strong opinions about the names here.

Reply:
The conventions seem to be related to GCM configurations. Let's leave the `x`-`y` coordinate names for now and we can change if/when we need.

Reply:
just delete the `standard_name` key then, since what you have isn't a standard name

Reply:
Done